tests/Oryx.Integration.Tests/Python/PythonEndToEndTests.cs

```diff
@@ -370,5 +370,101 @@ public async Task CanBuildAndRunPythonApp_UsingIntermediateDir_AndNestedOutputDi
                     Assert.Contains("Hello World!", data);
                 });
         }
+
+        [Fact]
+        [Trait("category", "python-3.11")]
```

Comment:
Should we add the same test for 3.12 as well?

Reply:
There's a test for 3.12 down below